is_prime.py

```diff
@@ -0,0 +1,9 @@
+def is_prime(n):
+    if n is None or not isinstance(n, int):
+        return False
+    if n <= 1:
+        return False
+    for i in range(2, int(n ** 0.5) + 1):
```

Comment:
Another readability preference, but I prefer to use `math.sqrt()` where I can, so the usage is more clear.